Pa3/Durchführung/Project/src/app/State/DriveOverGap.c

```diff
@@ -15,7 +15,8 @@
 #include "service/LineSensor.h"
 
 /* CONSTANTS **************************************************************************************/
-
+#define STATE_IS_ACTIVE_FOR_TOO_LONG 5000
+#define SENSOR_THRESHOLD 500
```

Comment:
DriveToStart uses the same Threshold. We should use something global:
CalibrateLineSensors.h:
 #define CALIB_NO_LINE(value) ((value) <= 500)
#define CALIB_OVER_LINE(value) ((value) > 500)